specification/search/data-plane/Azure.Search/preview/2023-07-01-Preview/searchindex.json

```diff
@@ -2381,6 +2411,10 @@
           "type": "string",
           "description": "The OData $filter expression to apply to the search query."
         },
+        "filterType": {
```

Comment:
Does this only apply to vector queries? Should it be a property of the vector object?